proto/bragi/bragi_service.proto

```diff
@@ -27,8 +28,12 @@ service Bragi {
 
 message MatchTimelineRequest {}
 
-message MatchTimelineResponse {
-  repeated Match matches = 1;
+message MatchTimelineMessage {
+  oneof message {
+    Matches matches = 1;
+    Match match = 2;
```

Comment:
`match` can be substituted by an array of matches with size 1. Probably it would be easier for client's integration that they would have just "one type of payload"

Reply:
Changed as discussed in the meeting